connman/include/log.h

```diff
@@ -60,8 +60,8 @@ struct connman_debug_desc {
 	const char *file;
 #define CONNMAN_DEBUG_FLAG_DEFAULT (0)
 #define CONNMAN_DEBUG_FLAG_PRINT   (1 << 0)
-//#define CONNMAN_DEBUG_FLAG_ALIAS   (1 << 1)
-//#define CONNMAN_DEBUG_FLAG_HIDE_NAME (1 << 2)
+#define CONNMAN_DEBUG_FLAG_ALIAS   (1 << 1)
+#define CONNMAN_DEBUG_FLAG_HIDE_NAME (1 << 2)
```

Comment:
Intentional change?

Reply:
Needed because of https://github.com/sailfishos/connman/pull/72/commits/a524e73ec70b72acbc7dabf2c5f2ea033d52ad8f - does not build without these. No idea why they were disabled in the first place.